### PR TITLE
Feature/swap to googledocstringspt1

### PR DESCRIPTION
**Pull request by kfezer**
Starting swapping docstrings to google style

### CODE REVIEW

**Comment by kfezer** [comment]
This can wait until we know it works with your documentation. I don't want to break that

**Comment by kfezer** [comment]
This should be all of them at this point

File: mycroft/engines/intent_engine.py [hunk 1]
```diff
@@ -49,17 +49,19 @@ def __init__(self, path_manager):
...
     @abstractmethod
     def calc_intents(self, query):
         """
         Run the intent engine to determine the probability of each intent against the query
-        :param query: input sentence as a single string
-        :return: dict of intent: intent_data where
+        Args:
+            query (str): input sentence as a single string
+        Returns:
+            intent (dict): intent_data where
```
Comment (MatthewScholefield) [inline mycroft/engines/intent_engine.py]: I wrote this weirdly the first time. I don't know how best to write this, but something like:
`dict of things where key is intent name and value is intent_data`. If you find a better way to write it, feel free to, but not a big deal.

**Review by MatthewScholefield — APPROVED** [review]
You can try to rebase and merge, but there might be a small conflict because `get_logger` is now gone.

File: mycroft/engines/intent_engine.py (hunk 1 above)
Comment (kfezer) [reply mycroft/engines/intent_engine.py]: Ok, I was wondering about that one and then skipped it until I could figure out what it was doing.
I can try something.

**Comment by MatthewScholefield** [comment]
lgtm

**Pull request merged**